solidity/app/cl_utils.js

```diff
@@ -0,0 +1,35 @@
+const clWallet = require('./cl_wallet.js')
+
+const Eth = require('ethjs')
+
+global.clUtils = global.clUtils || {
+  personalAccount: '0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f',
+  toWei: function (eth) {
+    return (parseInt(eth.toString(), 10) * 10 ** 18).toString()
+  },
+  getTxReceipt: function (txHash) {
+    return new Promise(async (resolve, reject) => {
+      for (let i = 0; i < 1000; i++) {
+        let receipt = await clUtils.eth.getTransactionReceipt(txHash)
+        if (receipt != null) {
+          return resolve(receipt)
+        }
+      }
+      reject(`${txHash} unconfirmed!`)
+    })
+  },
+  setProvider: function (provider) {
+    clUtils.provider = provider
+    clUtils.eth = new Eth(provider)
```

Comment:
In `clWallet` we use `this`, while here we reference `clUtils`. For the sake of consistency, should we also use `this` where `clUtils` is explicitly invoked in this file? This happens a few times in this file.